src/uncrustify_emscripten.cpp

```diff
@@ -612,1174 +588,629 @@ intptr_t _debug(intptr_t _file, lang_flag_e langIDX)
-      .value(STRINGIFY(CT_FPAREN_CLOSE), CT_FPAREN_CLOSE)
-      .value(STRINGIFY(CT_TPAREN_OPEN), CT_TPAREN_OPEN)
-      .value(STRINGIFY(CT_TPAREN_CLOSE), CT_TPAREN_CLOSE)
-      .value(STRINGIFY(CT_BRACE_OPEN), CT_BRACE_OPEN)
-      .value(STRINGIFY(CT_BRACE_CLOSE), CT_BRACE_CLOSE)
-      .value(STRINGIFY(CT_VBRACE_OPEN), CT_VBRACE_OPEN)
-      .value(STRINGIFY(CT_VBRACE_CLOSE), CT_VBRACE_CLOSE)
-      .value(STRINGIFY(CT_SQUARE_OPEN), CT_SQUARE_OPEN)
-      .value(STRINGIFY(CT_SQUARE_CLOSE), CT_SQUARE_CLOSE)
-      .value(STRINGIFY(CT_TSQUARE), CT_TSQUARE)
-      .value(STRINGIFY(CT_MACRO_OPEN), CT_MACRO_OPEN)
-      .value(STRINGIFY(CT_MACRO_CLOSE), CT_MACRO_CLOSE)
-      .value(STRINGIFY(CT_MACRO_ELSE), CT_MACRO_ELSE)
-      .value(STRINGIFY(CT_LABEL), CT_LABEL)
-      .value(STRINGIFY(CT_LABEL_COLON), CT_LABEL_COLON)
-      .value(STRINGIFY(CT_FUNCTION), CT_FUNCTION)
-      .value(STRINGIFY(CT_FUNC_CALL), CT_FUNC_CALL)
-      .value(STRINGIFY(CT_FUNC_CALL_USER), CT_FUNC_CALL_USER)
-      .value(STRINGIFY(CT_FUNC_DEF), CT_FUNC_DEF)
-      .value(STRINGIFY(CT_FUNC_TYPE), CT_FUNC_TYPE)
-      .value(STRINGIFY(CT_FUNC_VAR), CT_FUNC_VAR)
-      .value(STRINGIFY(CT_FUNC_PROTO), CT_FUNC_PROTO)
-      .value(STRINGIFY(CT_FUNC_START), CT_FUNC_START)
-      .value(STRINGIFY(CT_FUNC_CLASS_DEF), CT_FUNC_CLASS_DEF)
-      .value(STRINGIFY(CT_FUNC_CLASS_PROTO), CT_FUNC_CLASS_PROTO)
-      .value(STRINGIFY(CT_FUNC_CTOR_VAR), CT_FUNC_CTOR_VAR)
-      .value(STRINGIFY(CT_FUNC_WRAP), CT_FUNC_WRAP)
-      .value(STRINGIFY(CT_PROTO_WRAP), CT_PROTO_WRAP)
-      .value(STRINGIFY(CT_MACRO_FUNC), CT_MACRO_FUNC)
-      .value(STRINGIFY(CT_MACRO), CT_MACRO)
-      .value(STRINGIFY(CT_QUALIFIER), CT_QUALIFIER)
-      .value(STRINGIFY(CT_EXTERN), CT_EXTERN)
-      .value(STRINGIFY(CT_DECLSPEC), CT_DECLSPEC)
-      .value(STRINGIFY(CT_ALIGN), CT_ALIGN)
-      .value(STRINGIFY(CT_TYPE), CT_TYPE)
-      .value(STRINGIFY(CT_PTR_TYPE), CT_PTR_TYPE)
-      .value(STRINGIFY(CT_TYPE_WRAP), CT_TYPE_WRAP)
-      .value(STRINGIFY(CT_CPP_LAMBDA), CT_CPP_LAMBDA)
-      .value(STRINGIFY(CT_CPP_LAMBDA_RET), CT_CPP_LAMBDA_RET)
-      .value(STRINGIFY(CT_BIT_COLON), CT_BIT_COLON)
-      .value(STRINGIFY(CT_OC_DYNAMIC), CT_OC_DYNAMIC)
-      .value(STRINGIFY(CT_OC_END), CT_OC_END)
-      .value(STRINGIFY(CT_OC_IMPL), CT_OC_IMPL)
-      .value(STRINGIFY(CT_OC_INTF), CT_OC_INTF)
-      .value(STRINGIFY(CT_OC_PROTOCOL), CT_OC_PROTOCOL)
-      .value(STRINGIFY(CT_OC_PROTO_LIST), CT_OC_PROTO_LIST)
-      .value(STRINGIFY(CT_OC_GENERIC_SPEC), CT_OC_GENERIC_SPEC)
-      .value(STRINGIFY(CT_OC_PROPERTY), CT_OC_PROPERTY)
-      .value(STRINGIFY(CT_OC_CLASS), CT_OC_CLASS)
-      .value(STRINGIFY(CT_OC_CLASS_EXT), CT_OC_CLASS_EXT)
-      .value(STRINGIFY(CT_OC_CATEGORY), CT_OC_CATEGORY)
-      .value(STRINGIFY(CT_OC_SCOPE), CT_OC_SCOPE)
-      .value(STRINGIFY(CT_OC_MSG), CT_OC_MSG)
-      .value(STRINGIFY(CT_OC_MSG_CLASS), CT_OC_MSG_CLASS)
-      .value(STRINGIFY(CT_OC_M
```

Comment:
Is this code generated? Could it be?

Reply:
Yes, the enum values are updated with `update_emscripten_bindings.py`

Reply:
Okay. It would be nice if this could become a "properly" generated source file, then, i.e. have a build rule to generate it and have a "template" version in the repository that would be less "fluid". (But that could be a future enhancement.)